src/axolotl/utils/distributed.py

```diff
@@ -9,10 +9,29 @@
 import torch
 import torch.distributed as dist
 from accelerate import PartialState
+from transformers.utils.import_utils import (
+    is_torch_cuda_available,
+    is_torch_mps_available,
+    is_torch_npu_available,
+)
 
 distributed_state = None  # pylint: disable=invalid-name
 
 
+def get_device():
+    device = torch.device("cpu")
+    if is_torch_cuda_available():
+        device = torch.device("cuda")
+    elif is_torch_mps_available():
+        device = torch.device("mps")
+    elif is_torch_npu_available():
+        device = torch.device("npu")
+    return device
+
+
+CURRENT_DEVICE = get_device()
```

Comment:
I'm not a fan of this being a constant on this module

Reply:
fixed